packages/application-shell/src/components/fetch-project/fetch-project.js

```diff
@@ -35,9 +35,13 @@ export const mapAllAppliedToObjectShape = allAppliedShape =>
 class FetchProject extends React.Component {
   static displayName = 'FetchProject';
   static propTypes = {
-    projectKey: PropTypes.string.isRequired,
+    projectKey: PropTypes.string,
```

Comment:
This has to be optional after all now. The url doesn't have to have one. Never had to.